cmd/placement-translator/main.go

```diff
@@ -204,8 +193,8 @@ func main() {
 	mbwsPreInformer := rootInformerFactory.Tenancy().V1alpha1().Workspaces()
 	var _ tenancyv1a1informers.WorkspaceInformer = mbwsPreInformer
 
-	locationClusterPreInformer := edgeInformerFactory.Edge().V2alpha1().Locations()
-	var _ edgev1a1informers.LocationClusterInformer = locationClusterPreInformer
+	locationPreInformer := edgeInformerFactory.Edge().V2alpha1().Locations()
+	var _ edgev1a1informers.LocationInformer = locationPreInformer
```

Comment:
This line should have been deleted months ago, it was just a temporary thing to help write the code.

Reply:
Removed